lib/SIL/SILVerifier.cpp

```diff
@@ -1188,8 +1188,12 @@ class SILVerifier : public SILVerifierBase<SILVerifier> {
 
   // SWIFT_ENABLE_TENSORFLOW
   void checkGraphOperationInst(GraphOperationInst *GI) {
-    for (auto attr : GI->getAttributes())
+    llvm::DenseSet<Identifier> attributeNames;
+    for (auto attr : GI->getAttributes()) {
+      require(attributeNames.insert(attr.name).second,
```

Comment:
add a test case?

Reply:
There's no precedent of test-casing SIL verification cuz each `require` failure is a trap. There's only examples like testing whether `sil-opt` crashes or not (see [example](https://github.com/apple/swift/blob/master/test/SIL/verifier-fail-instruction.sil)). Doing so for every single `require` instruction is not practical. OTOH, IR verification is an implementation  detail anyway.